### PR TITLE
Catch Fedora-base image update problems early

### PR DESCRIPTION
**Pull request by cevich**
Previously updates were disabled due to the cloud VM only having 2-gig and the nested-VM only having 1-gig of memory.  Allow Fedora base-image package updates by increasing the available resources.  Enabling base-level (esp. kernel) package updates early supports spotting fundamental image problems early.  Otherwise they may not be found until a set of images is deployed downstream.

Also, update a few comments relating to followup package update.

### CODE REVIEW

**Comment by edsantiago** [comment]
- 20240809t163741z-f40f39d13
- 20240805t173227z-f40f39d13 &#x21D1; (built in #371)

|  |*debian*|*prior-fedora*|*fedora*|*fedora-aws*|*rawhide*|
|----|----|----|----|----|----|
|*kernel*       | 6.10.3-1        | 6.5.6-300       | 6.8.5-301       | 6.8.5-301       | 6.8.5-301       |
|               | 6.9.12-1&nbsp;&#x21D1; | 6.9.12-100&nbsp;&#x21D1; | 6.9.12-200&nbsp;&#x21D1; | 6.9.12-200&nbsp;&#x21D1; |                |
|*grub2-common* | 2.12-5          | 2.06-121        | 2.06-123        | 2.06-123        | 2.12-3          |
|               |                |                |                |                | 2.06-124&nbsp;&#x21D1; |
|*aardvark-dns* | 1.9.0-2         | 1.11.0-1        | 1.12.1-1        | 1.12.1-1        | 1.12.1-1        |
|               | 1.6.0-3&nbsp;&#x21D1; |                |                | 1.11.0-3&nbsp;&#x21D1; |                |
|*netavark*     | 1.9.0-4         | 1.11.0-1        | 1.12.1-1        | 1.12.1-1        | 1.12.1-1        |
|               | 1.6.0-2.1&nbsp;&#x21D1; |                |                | 1.11.0-3&nbsp;&#x21D1; | 1.11.0-3&nbsp;&#x21D1; |
|*gvisor-tap-vsock*| ?               | 0.7.4-1         | ?               | 0.7.4-1         | ?               |
|               |                | 0.7.3-1&nbsp;&#x21D1; |                | 0.7.3-2&nbsp;&#x21D1; |                |
|*passt*        | 2024-08-06      | 2024-06-24      | 2024-08-06      | 2024-07-26      | 2024-08-06      |
|               | 2024-07-26&nbsp;&#x21D1; |                | 2024-07-26&nbsp;&#x21D1; | 2024-06-24&nbsp;&#x21D1; | 2024-07-26&nbsp;&#x21D1; |
|*podman*       | 5.0.3+ds1-5+b1  | 4.9.4-1         | 5.2.0-1         | 5.1.2-1         | 5.2.0\~rc2-1    |
|               | 5.0.3+ds1-5&nbsp;&#x21D1; |                |                |                |                |
|*systemd*      | 256.4-2         | 254.16-1        | 255.10-3        | 255.10-3        | 256.4-1         |
|               |                |                | 255.10-1&nbsp;&#x21D1; | 255.10-1&nbsp;&#x21D1; |                |

File: .cirrus.yml [hunk 1]
```diff
@@ -48,7 +48,7 @@ image_builder_task:
     auto_cancellation: $CI != "true"
     stateful: true
     timeout_in: 50m
-    container:
+    container: &ibi_container
```
Comment (edsantiago) [inline .cirrus.yml:51]: `&ibi_container` is never used again. Is this a leftover from some previous iteration?

File: base_images/fedora_base-setup.sh [hunk 2]
```diff
@@ -35,15 +34,10 @@ if ! ((CONTAINER)); then
...
-# is available and debugging is also easier.  Try to save some more
-# memory by pre-populating repo metadata prior to any transactions.
-$SUDO dnf makecache $XARGS
-# Updates disable, see comment above
-# $SUDO dnf -y update $XARGS
-$SUDO dnf -y install $XARGS "${PKGS[@]}"
+$SUDO dnf makecache
+$SUDO dnf -y update
+$SUDO dnf -y install "${PKGS[@]}"
+$SUDO dnf -y update
```
Comment (edsantiago) [inline base_images/fedora_base-setup.sh:40]: This too looks like a leftover. If it is intentional, could you add a comment explaining what it's intended to do?

**Review by edsantiago — COMMENTED** [review]
LGTM with two maintainability issues

File: .cirrus.yml (hunk 1 above)
Comment (cevich) [reply .cirrus.yml:51]: Oops, meant to comment on this.  Yes, it's a leftover, I have a staged change to remove it.

File: base_images/fedora_base-setup.sh (hunk 2 above)
Comment (cevich) [reply base_images/fedora_base-setup.sh:40]: This one is on purpose, I'll add a comment.

**Review by edsantiago — APPROVED** [review]
LGTM

**Comment by edsantiago** [comment]
- 20240812t144740z-f40f39d13
- 20240805t173227z-f40f39d13 &#x21D1; (built in #371)

|  |*debian*|*prior-fedora*|*fedora*|*fedora-aws*|*rawhide*|
|----|----|----|----|----|----|
|*kernel*       | 6.10.3-1        | 6.5.6-300       | 6.8.5-301       | 6.8.5-301       | 6.8.5-301       |
|               | 6.9.12-1&nbsp;&#x21D1; | 6.9.12-100&nbsp;&#x21D1; | 6.9.12-200&nbsp;&#x21D1; | 6.9.12-200&nbsp;&#x21D1; |                |
|*grub2-common* | 2.12-5          | 2.06-121        | 2.06-123        | 2.06-123        | 2.12-4          |
|               |                |                |                |                | 2.06-124&nbsp;&#x21D1; |
|*aardvark-dns* | 1.9.0-2         | 1.11.0-1        | 1.12.1-1        | 1.12.1-1        | 1.12.1-1        |
|               | 1.6.0-3&nbsp;&#x21D1; |                |                | 1.11.0-3&nbsp;&#x21D1; |                |
|*netavark*     | 1.9.0-4         | 1.11.0-1        | 1.12.1-1        | 1.12.1-1        | 1.12.1-1        |
|               | 1.6.0-2.1&nbsp;&#x21D1; |                |                | 1.11.0-3&nbsp;&#x21D1; | 1.11.0-3&nbsp;&#x21D1; |
|*buildah*      | 1.35.3+ds1-3+b1 | 1.37.0-1        | 1.37.0-1        | 1.36.0-1        | 1.37.0-1        |
|               | 1.35.3+ds1-3&nbsp;&#x21D1; | 1.36.0-1&nbsp;&#x21D1; |                |                |                |
|*crun*         | 1.15-1+b1       | 1.15-1          | 1.15-1          | 1.15-1          | 1.15-2          |
|               | 1.15-1&nbsp;&#x21D1; |                |                |                |                |
|*gvisor-tap-vsock*| ?               | 0.7.4-1         | ?               | 0.7.4-1         | ?               |
|               |                | 0.7.3-1&nbsp;&#x21D1; |                | 0.7.3-2&nbsp;&#x21D1; |                |
|*passt*        | 2024-08-06      | 2024-06-24      | 2024-08-06      | 2024-07-26      | 2024-08-06      |
|               | 2024-07-26&nbsp;&#x21D1; |                | 2024-07-26&nbsp;&#x21D1; | 2024-06-24&nbsp;&#x21D1; | 2024-07-26&nbsp;&#x21D1; |
|*podman*       | 5.0.3+ds1-5+b2  | 4.9.4-1         | 5.2.0-1         | 5.1.2-1         | 5.2.0\~rc2-1    |
|               | 5.0.3+ds1-5&nbsp;&#x21D1; |                |                |                |                |
|*runc*         | 1.1.12+ds1-2+b1 | 1.1.12-1        | 1.1.12-3        | 1.1.12-3        | 1.1.12-4        |
|               | 1.1.12+ds1-2&nbsp;&#x21D1; |                |                |                |                |
|*skopeo*       | 1.13.3+ds1-2+b2 | 1.15.2-1        | 1.16.0-1        | 1.15.2-1        | 1.16.0-1        |
|               | 1.13.3+ds1-2+b1&nbsp;&#x21D1; |                |                |                |                |
|*systemd*      | 256.4-3         | 254.16-1        | 255.10-3        | 255.10-3        | 256.4-1         |
|               | 256.4-2&nbsp;&#x21D1; |                | 255.10-1&nbsp;&#x21D1; | 255.10-1&nbsp;&#x21D1; |                |

**Comment by cevich** [comment]
Force-push: rebased / resolved conflicts.

**Comment by edsantiago** [comment]
No changes in packages.

**Pull request merged**